prediction_market_agent/agents/microchain_agent/nft_treasury_game/app_nft_treasury_game.py

```diff
@@ -189,10 +201,11 @@ def show_about_agent_part(nft_agent: type[DeployableAgentNFTGameAbstract]) -> No
         else nft_agent.get_initial_system_prompt()
     )
     xdai_balance = get_balances(nft_agent.wallet_address).xdai
+    n_nft = BalanceOfNFT()(NFT_TOKEN_FACTORY, nft_agent.wallet_address)
     st.markdown(
         f"""### {nft_agent.name}
 
-Currently holds <span style='font-size: 1.1em;'><strong>{xdai_balance:.2f} xDAI</strong></span>.
+Currently holds <span style='font-size: 1.1em;'><strong>{xdai_balance:.2f} xDAI</strong></span> and still holds <span style='font-size: 1.1em;'><strong>{n_nft} NFT keys</strong></span>.
```

Comment:
Wondering if the `still holds` should be renamed to something like
```
f"still holds NFT keys" if balance == 1 else "does not hold NFT keys"
```

Reply:
Sure why not